src/store/messagesStore.js

```diff
@@ -501,6 +501,20 @@ const actions = {
 			})
 		}
 
+		if (message.systemMessage === 'call_started') {
+			context.dispatch('setConversationProperties', {
+				token: message.token,
+				properties: { callStartTime: message.timestamp },
+			})
+		}
+
+		if (message.systemMessage === 'call_ended_everyone') {
```

Comment:
```suggestion
		if ((message.systemMessage === 'call_ended' || message.systemMessage === 'call_ended_everyone')
		  && context.getters.conversation(token).lastActivity < message.timestamp) {
```
Should work for both cases